clarify certificate rotation docs (#6475)
aws/eks-anywhere: docs/content/en/docs/clustermgmt/security/manually-renew-certs.md

|                       | apiserver                |
|                       | front-proxy-client       |

You can rotate certificates by following the steps given below. You cannot rotate the `ca` certificate because it is the root certificate. Note that the commands used for Bottlerocket nodes are different than those for Ubuntu and RHEL nodes.

#### External etcd nodes

If your cluster is using external etcd nodes, you need to renew the etcd node certificates first. 

{{% alert title="Note" color="primary" %}}
You can check for external etcd nodes by running the following command:
```bash
kubectl get etcdadmcluster -A
```
{{% /alert %}}

> 1. SSH into each etcd node and run the following commands. Etcd automatically detects the new certificates after the `etcdadm join` command completes.

Reviewer: ```suggestion
1. SSH into each etcd node and run the following commands. Etcd automatically detects the new certificates and deprecates its old certificates.
```
since Etcd is detecting new certificates before `etcdadm join` command
